workers/unity/Packages/com.improbable.gdk.tools/GenerateFiles.cs

```diff
@@ -0,0 +1,37 @@
+using System.IO;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public class GenerateFiles
+    {
+        private static readonly string
+            ResourcesDir = Path.GetFullPath(Path.Combine(Application.dataPath, "Resources"));
```

Comment:
This pollutes the user root `Assets` directory - can we make this configurable?

Reply:
We can, however we need to have it stored in a Resources folder, I believe, to ensure it gets bundled in the player

Reply:
Sure, we can traverse up the directory tree and check that we are in a Resources folder (even on validation of the input).